br_nfe/models/invoice_eletronic.py

```diff
@@ -669,7 +682,7 @@ def _prepare_eletronic_invoice_values(self):
             'modFrete': self.modalidade_frete,
             'transporta': {
                 'xNome': self.transportadora_id.legal_name or
-                self.transportadora_id.name or '',
+                         self.transportadora_id.name or '',
```

Comment:
br_nfe/models/invoice_eletronic.py:685:26: E131 continuation line unaligned for hanging indent